webplugin/css/app/km-rich-message.css

```diff
@@ -1187,4 +1187,47 @@
 }
 .km-cta-multi-button-container .km-quick-replies.km-carousel-card-button {
     border-color: #e7e4e4
-}
\ No newline at end of file
+}
+.mck-form-template-wrapper input {
+    width: 93%;
+    padding: 9px;
+    margin-bottom: 2px;
+    display: inline-block;
+    border: none;
+    background-color: #ffffff;
+    color: #000000;
+    font-size: 10px;
+    margin-bottom: 12px;
+}
+.mck-form-template-wrapper label {
+    margin-bottom: 2px;
+    margin-left: 2px;
+}
+.mck-form-template-wrapper input::placeholder {
```

Comment:
@jithinbalu please add all the prefixes fr placeholder for different browsers as well. See this link: https://css-tricks.com/almanac/selectors/p/placeholder/

Reply:
changes are done